frontend/src/metabase/visualizations/visualizations/CartesianChart/events.ts

```diff
@@ -174,9 +185,58 @@ export const getEventColumnsData = (
       };
     })
     .filter(isNotNull);
+
+  return dataPoints;
 };
 
-export const getStackedTooltipModel = (
+const getTooltipFooterData = (
+  chartModel: BaseCartesianChartModel,
+  display: string,
+  seriesIndex: number,
+  dataIndex: number,
+): DataPoint[] => {
+  if (display === "scatter" || !isTimeSeriesAxis(chartModel.xAxisModel)) {
+    return [];
+  }
+
+  const datum = chartModel.dataset[dataIndex];
+  const seriesModel = chartModel.seriesModels[seriesIndex];
+
+  const currentValue = datum[seriesModel.dataKey];
+  const currentDate = dayjs(parseTimestamp(datum[X_AXIS_DATA_KEY]));
```

Comment:
since `parseTimestamp` returns dayjs objects we can drop `dayjs()` around it

Reply:
Whoops thanks for the heads up, I'll remove the `dayjs`